src/feed-parser.js

```diff
@@ -0,0 +1,20 @@
+const FeedParser = require('feedparser');
+const bent = require('bent'); // for fetching the feed
+
+module.exports = async function(feed){
+    const feedparser = new FeedParser();
+    const getStream = bent(feed);
+
+    let stream = await getStream('/json.api');
+    return new Promise((res,rej)=>{
+        stream.pipe(feedparser)
+            .on('error', rej)
+            .on('readable', function(){
```

Comment:
Change this to just be `on('readable', resolve);` and return the entire `item` that's being read.